rollup.config.js

```diff
@@ -0,0 +1,28 @@
+import buble from 'rollup-plugin-buble';
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import conditional from 'rollup-plugin-conditional';
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+export default {
```

Comment:
If you do not want to have to write `"use strict";` at the top of every file, rollup can be configured to do this automatically for you: https://github.com/rollup/rollup/wiki/JavaScript-API#usestrict